packages/auth-remix/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@edgedb/auth-remix",
+  "description": "Helper library to integrate the EdgeDB Auth extension with Remix",
+  "version": "0.1.0-alpha.1",
+  "author": "EdgeDB <info@edgedb.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/edgedb/edgedb-js.git",
+    "directory": "packages/auth-remix"
+  },
+  "license": "Apache-2.0",
+  "sideEffects": false,
+  "files": [
+    "/dist"
+  ],
+  "scripts": {
+    "build": "tsc --project tsconfig.json"
+  },
+  "devDependencies": {
+    "@types/node": "^20.8.4",
+    "edgedb": "^1.3.6",
+    "tailwindcss": "^3.3.5",
+    "typescript": "^5.2.2"
+  },
+  "peerDependencies": {
+    "edgedb": "^1.3.6"
+  },
+  "dependencies": {
+    "@edgedb/auth-core": "^0.1.0-beta.1",
+    "@octokit/core": "^5.0.2",
+    "@remix-run/node": "^2.3.1",
```

Comment:
```suggestion
```
I don't think either of these deps are needed now either.

Reply:
hey I use redirect & json from @remix-run/node, but deleted `@remix-run/server-runtime` since I don't use it anywhere anymore

Reply:
ah I saw your comment below about using @remix-run/server-runtime instead of /node, will check it and update